src/test/java/com/fryrank/controller/ReviewControllerTests.java

```diff
@@ -100,4 +103,106 @@ public void testAddNewReviewNullBody() throws Exception {
 
         controller.addNewReviewForRestaurant(expectedReview);
     }
+
+    // /api/reviews/aggregateInformation endpoint tests
+    @Test
+    public void testGetSingleRestaurantAllAggregateInformation() throws Exception {
+        final Map<String, AggregateReviewInformation> expectedAggregateReviewInformation = Map.of(
+                "ChIJl8BSSgfsj4ARi9qijghUAH0", new AggregateReviewInformation("ChIJl8BSSgfsj4ARi9qijghUAH0", 5.0F)
```

Comment:
It's a good idea to make the input values constants that can be reused when we're validating later